indexing-hadoop/src/main/java/io/druid/indexer/hadoop/HadoopWritableInputRow.java

```diff
@@ -0,0 +1,10 @@
+package io.druid.indexer.hadoop;
+
+import io.druid.data.input.InputRow;
+import org.apache.hadoop.io.Writable;
+
+/**
+ */
+public interface HadoopWritableInputRow extends InputRow, Writable
```

Comment:
This only has to be writable if it is passing between the Mapper and Combiner/Reducer.  It is legitimate and ok to pass non Writable POJOs as input to the Mapper and as output from the Reducer (the contract is between InputFormat and Mapper, or OutputFormat and Reducer).  So, I don't believe we actually need a "writable" InputRow.